temporal-sdk/src/main/java/io/temporal/activity/ActivityOptions.java

```diff
@@ -83,37 +83,57 @@ private Builder(ActivityOptions options) {
     }
 
     /**
-     * Overall timeout workflow is willing to wait for activity to complete. It includes time in a
-     * task queue (use {@link #setScheduleToStartTimeout(Duration)} to limit it) plus activity
-     * execution time (use {@link #setStartToCloseTimeout(Duration)} to limit it). Either this
-     * option or both schedule to start and start to close are required.
+     * Overall timeout workflow is willing to wait for activity to complete.
+     *
+     * <p>ScheduleToCloseTimeout limits total time of the activity execution including retries (use
+     * StartToCloseTimeout to limit a time of a single attempt).
+     *
+     * <p>Either this option or StartToClose are required.
+     *
+     * <p>Defaults to unlimited.
```

Comment:
It actually is not how it works right now. It looks like there is no unit test for such case and it should be. If you try to create an activity with ActivityOptions.newBuilder().validateAndBuildWithDefaults() it will fail with GRPC message saying that either scheduleToClose or startToClose have to be defined.